lib/transcoder.rb

```diff
@@ -27,7 +27,7 @@ def transcode(output:, format:)
     encoding_options = {}
     case format
     when HE_AAC
-      encoding_options[:audio_codec] = 'libfdk_aac'
+      encoding_options[:audio_codec] = ENV['AAC_CODEC']
```

Comment:
Should we default it to `libfdk_aac` like this to make this change backwards compatible?:

`encoding_options[:audio_codec] = ENV['AAC_CODEC'] || 'libfdk_aac'`